packages/components/src/elevation/types.ts

```diff
@@ -8,19 +8,19 @@ export type Props = {
 	/**
 	 * Renders the active (interaction) shadow value.
 	 */
-	active?: number;
+	active?: number | null;
```

Comment:
Should we add `@default` tags matching the `DEFAULT_PROPS` defined in `packages/components/src/elevation/component.tsx` ?